app/next-i18next.config.js

```diff
@@ -2,10 +2,5 @@ module.exports = {
   i18n: {
     defaultLocale: "en",
     locales: ["en", "es"],
-    react: {
```

Comment:
It's poorly documented but it looks like the fix, while preserving the safelisting of `em`, is to unnest this `react` object outside of the `i18n` object:

<img width="798" alt="CleanShot 2022-10-19 at 09 25 49@2x" src="https://user-images.githubusercontent.com/371943/196750435-786d4bb3-337d-412c-b8f4-24d6f4bb6454.png">

Reply:
I know you've removed your blocking review, just for the record, I'm fine with this approach too; it's one of those cases where I don't quite understand why `em` isn't included by default, so it's just a question of whether to go with the default because it's the default, or with something else that seems to make more sense. Since it's not likely to matter either way, I'm still thinking we stick with the default in the template. I'll leave this open for another day or so in case Rocket wants to chime in, and then I'll close this out.